lang/regina-rexx/Makefile.inc

```diff
@@ -0,0 +1,26 @@
+COMMENT=  	Regina REXX scripting language
+
+DISTNAME=      	Regina-REXX-${VERSION}
+PKGNAME =       regina-rexx-${VERSION}
+
+#SHARED_ONLY=    Yes
+
+CATEGORIES=     lang
+
+MASTER_SITES=  http://jaist.dl.sourceforge.net/sourceforge/regina-rexx/ \
+                http://voxel.dl.sourceforge.net/sourceforge/regina-rexx/ \
+                http://internode.dl.sourceforge.net/sourceforge/regina-rexx/ \
+                http://fastbull.dl.sourceforge.net/sourceforge/regina-rexx/ \
+                http://freefr.dl.sourceforge.net/sourceforge/regina-rexx/ 
```

Comment:
I think that could be replaced with <code>MASTER_SITES= ${MASTER_SITE_SOURCEFORGE:=regina-rexx/}</code>.